spec/entrypoint_spec.sh

```diff
@@ -0,0 +1,25 @@
+#shellcheck shell=sh
```

Comment:
This is a good start but for completeness you could add a golden image that has a known footprint of vulnerabilities like this https://hub.docker.com/r/knqyf263/vuln-image

Then you can have more concrete assertions.